sympy/physics/units.py

```diff
@@ -243,38 +241,8 @@ def free_symbols(self):
 g = gram = grams = kilogram / kilo
 mg = milligram = milligrams = milli * g
 ug = microgram = micrograms = micro * g
-
-
-#----------------------------------------------------------------------------
-# Physical constants
-#
-c = speed_of_light = 299792458 * m/s
-G = gravitational_constant = Rational('6.67408') * ten**-11 * m**3 / kg / s**2
-u0 = magnetic_constant = 4*pi * ten**-7 * N/A**2
-e0 = electric_constant = 1/(u0 * c**2)
-Z0 = vacuum_impedance = u0 * c
-
-planck = Rational('6.62607004') * ten**-34 * J*s
-hbar = planck / (2*pi)
-
-avogadro_number = Rational('6.022140857') * 10**23
-avogadro = avogadro_constant = avogadro_number / mol
-boltzmann = Rational('1.38064852') * ten**-23 * J / K
-atomic_mass_constant = Rational('1.660539040') * ten**-27 * kg
-
-gee = gees = Rational('9.80665') * m/s**2
-atmosphere = atmospheres = atm = 101325 * pascal
-
-kPa = kilo*Pa
-bar = bars = 100*kPa
-pound = pounds = 0.45359237 * kg * gee  # exact
-psi = pound / inch ** 2
-dHg0 = 13.5951  # approx value at 0 C
-mmHg = dHg0 * 9.80665 * Pa
 amu = amus = gram / avogadro / mol
 mmu = mmus = gram / mol
-quart = quarts = Rational(231, 4) * inch**3
-eV = Rational('1.6021766208') * ten**-19 * J
 
```

Comment:
atmosphere, kPa, bar, pound, psi, mmHg, amu, quart, eV are all units, you can look at the quantities package and follow the grouping there between units and constants.